db/data.py

```diff
@@ -265,17 +265,15 @@ def set_model_status(model_name, model_version, model_status):
                             "model_status": model_status})
 
 
-def get_model(model_name, model_version):
```

Comment:
this method wasn't used anywhere, so I hijacked the implementation to make a method that we needed